src/main/resources/META-INF/persistence.xml

```diff
@@ -6,6 +6,15 @@
 
 	<persistence-unit name="default" transaction-type="RESOURCE_LOCAL">
 		<provider>org.hibernate.ejb.HibernatePersistence</provider>
+		<class>nl.tudelft.ewi.devhub.server.database.entities.BuildResult</class>
```

Comment:
If you ever want to add, remove, or rename entities you have to update this file. Hence autodiscover is easier and less error-prone for devs.